images/win/scripts/Installers/Install-Firefox.ps1

```diff
@@ -1,16 +1,24 @@
 ################################################################################
 ##  File:  Install-Firefox.ps1
 ##  Desc:  Install Mozilla Firefox
+##  Supply chain security: Firefox browser - checksum validation
 ################################################################################
 
 # Install and configure Firefox browser
 Write-Host "Install latest Firefox browser..."
 $VersionsManifest = Invoke-RestMethod "https://product-details.mozilla.org/1.0/firefox_versions.json"
-$InstallerName = "firefox-browser.exe"
 $InstallerUrl = "https://download.mozilla.org/?product=firefox-$($VersionsManifest.LATEST_FIREFOX_VERSION)&os=win64&lang=en-US"
 $ArgumentList = ("/silent", "/install")
```

Comment:
It may be better to place this right before installation invocation. What do you think?